README.md

```diff
@@ -13,6 +13,18 @@ This workflow also utilizes `update-artifact-file` workflow, to edit the `artifa
 | repository | If you want to override default's docker image name                       | string | no        |               |
 | context    | From where the CI should build the docker image                           | string | no        | . (Current context)             |
 
+```mermaid
+flowchart TD
+    classDef head fill:#5882FA
+    classDef workflow fill:#1E4E20
+    A[Build And Push Docker]:::head --> B[Checkout latest commit]
+    B --> C(Login to Remote Registry)
+    C --> D(Generate Docker Image Name)
+    D --> E( Build Image)
+    E --> F[Push Image]
+    A --> G[workflow: Edit artifacts.json in helm-charts]:::workflow
```

Comment:
Change to "Trigger update-artifacts-file workflow".
Plus, arrow should come from `step E`, since it would be triggerd **only if** first job is completed.

Reply:
Fixed